### PR TITLE
updates for harvester-seeder to support webhook

### PR DESCRIPTION
**Pull request by ibrokethecloud**
PR introduces changes for harvester-seeder to support newly introduced webhook for validating inventory objects.

harvester-seeder PR dependent on this: https://github.com/harvester/seeder/pull/17

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -56,6 +56,10 @@ jobs:
         uses: helm/kind-action@v1.2.0
         if: steps.list-changed.outputs.changed == 'true'
 
+      - name: Create harvester-system namespace
+        run: "kubectl create namespace harvester-system"
+        if: steps.list-changed.outputs.changed == 'true'
+
```
Comment (guangbochen) [inline .github/workflows/ci.yaml:62]: can we cherry-pick the commit, thanks.

Comment (ibrokethecloud) [reply .github/workflows/ci.yaml:62]: done.

**Review by guangbochen — APPROVED** [review]
LGTM, thanks.

**Review by bk201 — APPROVED** [review]
LGTM!

**Review by starbops — APPROVED** [review]
Tested with the following scenarios along with harvester/seeder#17 and LGTM!

1. Configure one node with BMC A then edit it back to BMC B, seeder can correctly do power action against the right node.
2. Configure the first node's OOB info, then configure the second node's OOB info as same as the first node's, seeder webhook rejects the request correctly.

The above tests are done via virtualbmc and real hardware BMCs. And the session logout fix for Redfish is also working (tested with Fremont machines since my bare-metal doesn't have a valid license for Redfish). Thanks!

**Pull request merged**